app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -402,6 +406,30 @@ public void testBacktrack() {
     }
 
     /**
+     * verifyTrustworthiness test
+     * Tests whether a block is trustworthy
+     */
+    @Test
+    public void testVerifyTrustworthiness() {
+        final Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, owner, bc.getLatestSeqNumber(owner)+1,
```

Comment:
In the blockcontroller class, when you create a block, the value contact of who the block is, is needed as well. However, we are not using it anywhere else, not even to calculate the value of previousHashSender.